### ISSUE TITLE
Hyperlink in error message in web UI goes to wrong line (fragment ID doesn’t match line number it actually links to)

### ISSUE THREAD

**Issue by Sivustonikkari** [validator/validator#1574]
at https://vapaa-ajattelijat.fi the validation gives the following error:

```
Error: Duplicate attribute src.
At line 900, column 135
ljett.jpg"  src="https://vapaa
```
But when looking at the source, there is only on src element.

**Comment by sideshowbarker** [comment]
The `img` element in the source at line 900 does have a duplicate `src` attribute — the problem is that the https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fvapaa-ajattelijat.fi%2F#cl900c135 link in the web UI unexpectedly goes instead to the `img` element at line 937.

In other words, the checker is reporting the correct line number in the error message, and has that same line number int the fragment part of the hyperlink URL in the error message — but that fragment ID doesn’t match the line number it actually links to.

**Comment by j9t** [comment]
Does this issue still persist?

When I’m checking https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fvapaa-ajattelijat.fi%2F, the line numbers and links seem to all match up.

**Comment by Sivustonikkari** [comment]
> Does this issue still persist?
> 
> When I’m checking https://validator.w3.org/nu/?showsource=yes&doc=https%3A%2F%2Fvapaa-ajattelijat.fi%2F, the line numbers and links seem to all match up.

Seems to be fine now.

**Comment by j9t** [comment]
Thanks! Closing this one but let’s reopen if this reappears.

FYI @sideshowbarker
